src/basics/Text.js

```diff
@@ -232,7 +248,7 @@ export const Bold = styled.strong`
 export const Muted = styled(Text).attrs(() => ({ as: "span" }))`
   color: ${({ theme }) => theme.medium};
 `;
-export const Strike = styled(Text).attrs(() => ({ as: "del" }))`
+export const Strike = styled(Text)`
```

Comment:
Do we want to remove the `del` tag? I think that helps e.g. screenreaders identify the content.